modules/servers/upCloudVps/lib/configOptions.php

```diff
@@ -37,13 +37,6 @@ public function configs(){
     ];
   }
 
-  private function getNetworking()

```

Comment:
There is a commented out call to this function a few lines above, on line 36. Let's remove that too.